pandas/core/arrays/base.py

```diff
@@ -869,11 +871,12 @@ def view(self, dtype=None) -> Union[ABCExtensionArray, np.ndarray]:
         Parameters
         ----------
         dtype : str, np.dtype, or ExtensionDtype, optional
-            Default None
+            Default None.
 
         Returns
         -------
         ExtensionArray
+            A view of the :class:`ExtensionArray` calling :meth:`view`.
```

Comment:
I don’t think the :meth: resolves to anything.

Reply:
thanks, It looked to me as though that functionality was not fully built out so I just tried to describe it as literally as possible in order to pass the checks. Would ```A view of the :class:`ExtensionArray```` be sufficient?

Reply:
yes, I think removing the `calling...` is good enough, thanks @HughKelley